lms/js_config_types.py

```diff
@@ -0,0 +1,27 @@
+from typing import NotRequired, TypedDict
+
+
+class APICallInfo(TypedDict):
+    path: str
+    authUrl: NotRequired[str]
+
+
+class APIAssignment(TypedDict):
+    id: int
+    title: str
+
+
+class APIAssignmentStats(TypedDict):
+    display_name: str
+    annotations: int
+    replies: int
+    last_activity: str
+
+
+class DashboardLinks(TypedDict):
+    assignmentApi: str
+    assignmentStatsApi: str
+
+
+class DashboardConfig(TypedDict):
+    links: DashboardLinks
```

Comment:
Perhaps we should use the terminology "routes", since that is what we call them in the backend. We would reserve "links" for things that you can actually open in the browser.

Reply:
Changed to `routes`